interfaces/effect_player.hpp

```diff
@@ -117,6 +117,17 @@ namespace bnb::oep::interfaces
          * @example draw()
          */
         virtual void draw() = 0;
+
```

Comment:
This should be under define and activated only in the case of Metal - else all other samples depending on that interface will be broken.

Reply:
Better remove this change to interfaces and move it to the sample as a specific.